### ISSUE TITLE
logfire with distributed package like CLI

### ISSUE THREAD

**Issue by grll** [pydantic/logfire#651]
### Question

Hello, I would like to use logfire to collect logs from a distributed pip package that acts as a CLI. 

The issue is that for now the only easy method I see to log into my logfire project would be to share the write_token to everyone through the pip package. Is there a way you would recommend to go differently about that ? 

An authentication system could be put in place but then I would need to be able to create write_token programmatically for each new user, is it something you facilitate ? Or do you have other suggestions?

**Comment by alexmojaki** [comment]
Yes, you can do that. The SDK already creates write tokens programmatically using a higher level of authentication called 'user tokens':

https://github.com/pydantic/logfire/blob/743fb5d720ffa8b6b99d3f53a56d54bf10c5d8d5/logfire/_internal/config.py#L1226-L1227

https://github.com/pydantic/logfire/blob/743fb5d720ffa8b6b99d3f53a56d54bf10c5d8d5/logfire/_internal/config.py#L1296-L1301

So if you keep a user token secret you can use it to generate write tokens, give those out privately, and suspend them if they're abused.

**Comment by alexmojaki** [comment]
Closing because it seems like the question has been answered and I don't think we want to do anything more here for now. @grll please let us know how this goes, whether good or bad.
